### ISSUE TITLE
Актуальность данных

### ISSUE THREAD

**Issue by projct1** [xmlcalendar/data#34]
Добрый день.

Откуда вы берёте данные и на сколько они соответствуют производственному календарю КонсультантПлюс? https://www.consultant.ru/law/ref/calendar/proizvodstvennye/2022/?

**Comment by projct1** [comment]
На данный момент вижу, что для 2022 нету перенесённых дней, как на консультанте:
- с субботы 1 января на вторник 3 мая;
- с воскресенья 2 января на вторник 10 мая;
- с субботы 5 марта на понедельник 7 марта.

**Comment by xmlcalendar** [comment]
Спасибо, что зарепортили проблему.
Данные должны соответствовать производственному календарю КонсультантПлюс.
Да это ошибка, пропали переносы за 2022 год, в ближайшее время будет исправлено.

**Comment by projct1** [comment]
Спасибо. А вы данные берёте с КонсультантПлюс? Парсите?
И когда появится календарь за 2023?

**Comment by xmlcalendar** [comment]
Есть парсер, который достает данные с lugasoft, но доверять им нельзя, поэтому они сравниваются с КонсультантПлюс и Гарант (и не только). Парсера который бы доставал российский календарь с этих сайтов пока нет.
Иногда пользователи присылают календари на почту или создают PR в github.

Обычно календари на следующий год появляются в августе. из-за того что документы до этого срока обычно уже утверждены.

**Comment by projct1** [comment]
Понятно) То есть самая актуальная инфа на КонсультантПлюс?

**Comment by xmlcalendar** [comment]
Самая актуальная информация на сайте [kremlin.ru](http://www.kremlin.ru/) в виде указов президента. К примеру приходит указ перенести день с такого-то по такой. Консультант ссылаясь на указ обновляет свой календарь.
Тоже самое делают другие поставщики календаря. 
На текущий момент на сайте КонсультантПлюс есть производственный календарь, но т.к. президент РФ еще его не утвердил, то этот календарь не имеет большого смысла (т.е. это возможный календарь на 2023 год). Сделан он для того чтобы попасть в индекс поисковиков, чтобы КонсультантПлюс смог заработать на этом деньги с переходов. Поэтому появляется он обычно гораздо раньше.

**Comment by xmlcalendar** [comment]
Обновлены отметки о переносах: http://xmlcalendar.ru/data/ru/2022/calendar.xml

**Comment by projct1** [comment]
> Самая актуальная информация на сайте [kremlin.ru](http://www.kremlin.ru/) в виде указов президента. К примеру приходит указ перенести день с такого-то по такой. Консультант ссылаясь на указ обновляет свой календарь. Тоже самое делают другие поставщики календаря. На текущий момент на сайте КонсультантПлюс есть производственный календарь, но т.к. президент РФ еще его не утвердил, то этот календарь не имеет большого смысла (т.е. это возможный календарь на 2023 год). Сделан он для того чтобы попасть в индекс поисковиков, чтобы КонсультантПлюс смог заработать на этом деньги с переходов. Поэтому появляется он обычно гораздо раньше.

Да, только на консультанте удобно всё это выглядит, ведь на кремлине нету календаря. Консультант заинтересован поддерживать актуальность информации.
